extlinks/links/management/commands/upload_all_archived_linkevents.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+from extlinks.common.management.commands import BaseCommand
+from django.core.management import call_command
+
+
+class Command(BaseCommand):
+    help = "Uploads all archives currently located in the backup directory to object storage"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--dir",
+            help="The directory from which to upload archives.",
+            type=str
+        )
+
+    def _handle(self, *args, **options):
+        path = options['dir']
+        for filename in os.listdir(path):
+            if filename.endswith('.json.gz'):
```

Comment:
If we want this to be linkevent specific, then I would do
```suggestion
            if filename.endswith('.json.gz') and filename.startswith('links_linkevent_'):
```
to make sure we don't pickup future things like aggregates.
Otherwise, we should probably change the language to just reference archives rather than linkevent archives specifically.

Reply:
Great catch!